Services/IMenuService.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace DataManager.Services
+{
+    internal interface IMenuService
```

Comment:
ustaw sposob dostepnośći do interfejsu na public. Przydałyby sie metody które beda wywolywane z klasy MenuService.cs ale to juz potem zrobimy jako todo